my-app/src/Components/Navigation.tsx

```diff
@@ -8,21 +8,28 @@ import { PostPage } from "./PostPage";
 import { ChangeThemeContext, ThemeContext } from "./Context/themeContext";
 import { SearchPostsList } from "./SearchPostsList";
 import { Header } from "./Header";
+import { Routes, Route, useNavigate } from "react-router-dom";
 
 type User = {
-   login: string | null,
-   pass: string | null,
-   username: string | null
+   login?: string,
+   pass?: string,
+   username?: string
+}
+
+const TEST_USER = {
+   login: 'user',
+   pass: 'user',
+   username: 'Artem Malkin'
 }
 
 export const Navigation = () => {
    const changeTheme = useContext(ChangeThemeContext);
    const theme = useContext(ThemeContext);
 
-   const [users, setUsers] = useState<User[]>([])
+   const [users, setUsers] = useState<User[]>([TEST_USER])
 
    const [page, setPage] = useState('auth');
-   const [user, setUser] = useState<User>({ login: null, pass: null, username: null });
+   const [user, setUser] = useState<User>({});
```

Comment:
the best place is in the context (or custom hook)
example https://github.com/remix-run/react-router/blob/dev/examples/auth/src/App.tsx